backend/import/xml/parser.js

```diff
@@ -116,6 +117,12 @@ const getTrueType = (type) => (
   type.split("-")[0].trim()
 );
 
+const addManaCostBrackets = (manacost) => {
+  let manaSymbolRegExp = /(?:(?<!\/)(?:\d+|[^/\s])(?!\/))|(?:\S\/\S)/g;
```

Comment:
This should be a constant defined once outside the function

Reply:
Aha, you're right, thanks!